index.html

```diff
@@ -146,7 +146,7 @@ <h2 class="content-title">Multi-Post Stories</h2>
             </div>
         </article>
     </section>
-    <section id="about-section" tabindex="20">
```

Comment:
The "tabindex" attribute was removed to enable users to navigate through the structure of the website using the tab key. without hitch-free